nbconvert/preprocessors/execute.py

```diff
@@ -202,14 +212,13 @@ def preprocess(self, nb, resources):
 
         # from jupyter_client.manager import start_new_kernel
 
-        def start_new_kernel(startup_timeout=60, kernel_name='python',
```

Comment:
I think the intention of this function is that at some point it can be replaced with the `jupyter_client` import above. So let's leave this as is, and pass in `startup_timeout` where it's called a few lines below.